lib/core/datatable/components/datatable/datatable.component.html

```diff
@@ -31,10 +31,9 @@
                  role="columnheader"
                  [attr.tabindex]="showHeader ? 0 : null"
                  [attr.aria-sort]="col.sortable ? (getAriaSort(col) | translate) : null"
-                 title="{{ col.title | translate }}"
                  adf-drop-zone dropTarget="header" [dropColumn]="col">
-                <span *ngIf="col.srTitle" class="adf-sr-only">{{ col.srTitle | translate }}</span>
                 <span *ngIf="col.title" class="adf-datatable-cell-value">{{ col.title | translate}}</span>
+                <span class="adf-sr-only" aria-live="assertive">{{getAriaSort(col) !== 'ADF-DATATABLE.ACCESSIBILITY.SORT_NONE' ? (getAriaSort(col) | translate) + ' ' +  (col.title | translate) : ('ADF-DATATABLE.ACCESSIBILITY.SORT_DEFAULT' | translate)}}</span>
```

Comment:
can we have a method here? it's hard to read

Reply:
+1

Reply:
yep can do!